Libs/Visualization/VTK/Core/vtkScalarsToColorsContextItem.h

```diff
@@ -46,6 +46,8 @@ class CTK_VISUALIZATION_VTK_CORE_EXPORT vtkScalarsToColorsContextItem
 public:
   static vtkScalarsToColorsContextItem* New();
 
+  void SetLeftAxisMode(int mode);
```

Comment:
If there is a Set, there must be a Get.
Please add comments

Reply:
Done